### PR TITLE
Grouping chooser commit on change

### PR DESCRIPTION
**Pull request by leogretz2**
Hoist P/R Checklist
-------------------

**Pull request authors:** Review and check off the below. Items that do not apply can also be
checked off to indicate they have been considered. If unclear if a step is relevant, please leave
unchecked and note in comments.

- [x] Caught up with `develop` branch as of last change.
- [x] Added CHANGELOG entry, or determined not required.
- [x] Reviewed for breaking changes, added `breaking-change` label + CHANGELOG if so.
- [x] Updated doc comments / prop-types, or determined not required.
- [x] Reviewed and tested on Mobile, or determined not required.
- [x] Created Toolbox branch / PR, or determined not required.

**If your change is still a WIP**, please use the "Create draft pull request" option in the split
button below to indicate it is not ready yet for a final review.

> **Pull request reviewers:** when merging this P/R, please consider using a *squash commit* to
> collapse multiple intermediate commits into a single commit representing the overall feature
> change. This helps keep the commit log clean and easy to scan across releases. PRs containing a
> single commit should be *rebased* when possible.

### CODE REVIEW

File: cmp/grouping/GroupingChooserModel.ts [hunk 1]
```diff
@@ -51,6 +51,9 @@ export interface GroupingChooserConfig {
 
     /** Maximum number of dimensions allowed in a single grouping. */
     maxDepth?: number;
+
+    /** False (default) requires user to click outside Popover for table to update. */
```
Comment (amcclain) [inline cmp/grouping/GroupingChooserModel.ts:55]:
```suggestion
    /** False (default) waits for the user to dismiss the popover before updating the external/observable value. */
```

File: desktop/cmp/grouping/GroupingChooser.ts [hunk 2]
```diff
@@ -27,7 +27,7 @@ export interface GroupingChooserProps extends ButtonProps<GroupingChooserModel>
     popoverMinHeight?: number;
 
     /** Position for chooser popover, as per Blueprint docs. */
```
Comment (amcclain) [inline desktop/cmp/grouping/GroupingChooser.ts:29]:
```suggestion
    /** Position of popover relative to target button. */
```

**Review by amcclain — APPROVED** [review]
Two minor tweaks I'd suggest to the comments, but looking good to me!

**Review by amcclain — APPROVED** [review]
Two minor tweaks I'd suggest to the comments, but looking good to me!

**Comment by lbwexler** [comment]
@leogretz2 -- I wend ahead and merged this with Anselm's doc comment changes.   Please check out those changes -- definitely worth understanding his suggestions and the API docs style we are looking for.

**Pull request merged**